src/components/PeoplePage.tsx

```diff
@@ -1,31 +1,128 @@
+import { useState, useEffect } from 'react';
+import { useSearchParams } from 'react-router-dom';
 import { PeopleFilters } from './PeopleFilters';
-import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { Loader } from './Loader';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { SearchParams, getSearchWith } from '../utils/searchHelper';
 
 export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [errorMessage, setErrorMessage] = useState(false);
+  const [searchParams, setSearchParams] = useSearchParams();
+  const query = searchParams.get('query') || '';
+  const filterGender = searchParams.get('sex') || 'all';
+  const filterCenturies = searchParams.getAll('centuries') || [];
```

Comment:
You don't need it here. Put the logic for filtering `sex`, `query` and `centuries` inside `PeopleFilter` component